### PR TITLE
Add STRIDE tokens

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

File: blockchains/stride/assets/staevmos/info.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Stride Staked Evmos",
+    "type": "STRIDE",
+    "symbol": "STEVMOS",
+    "decimals": 6,
```
Comment (vcoolish) [inline blockchains/stride/assets/staevmos/info.json:5]: 18

File: blockchains/stride/assets/staevmos/info.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "website": "https://stride.zone",
+    "description": "Stride Staked Evmos is the token received in exchange for staking EVMOS with Stride.",
+    "explorer": "https://www.mintscan.io/stride",
+    "status": "active",
+    "id": "staevmos",
+    "tags": [
+        "staking-native"
```
Comment (vcoolish) [inline blockchains/stride/assets/staevmos/info.json:12]: staking-liquid

File: blockchains/stride/assets/stinj/info.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Stride Staked Injective",
+    "type": "STRIDE",
+    "symbol": "STINJ",
+    "decimals": 6,
```
Comment (vcoolish) [inline blockchains/stride/assets/stinj/info.json:5]: 18

File: blockchains/stride/assets/stuatom/info.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Stride Staked Atom",
+    "type": "STRIDE",
+    "symbol": "STATOM",
+    "decimals": 6,
+    "website": "https://stride.zone",
+    "description": "Stride Staked Atom is the token received in exchange for staking ATOM with Stride. STATOM is redeemable 1:1 for ATOM, subject to the 21 day unbonding period on Cosmos.",
+    "explorer": "https://www.mintscan.io/stride",
+    "status": "active",
+    "id": "statom",
```
Comment (vcoolish) [inline blockchains/stride/assets/stuatom/info.json:10]: stuatom

File: blockchains/stride/assets/staevmos/info.json (hunk 1 above)
Comment (doperiddle) [reply blockchains/stride/assets/staevmos/info.json:5]: Is there something I need to do?or will the stride tokens show up automatically.thank u so much

Comment (apepayer) [reply blockchains/stride/assets/staevmos/info.json:5]: no, its all automatic after merge

Comment (vcoolish) [reply blockchains/stride/assets/staevmos/info.json:5]: no, it's automatic after merge

Comment (doperiddle) [reply blockchains/stride/assets/staevmos/info.json:5]: Thank you so much!!!!!!!

**Review by doperiddle — COMMENTED** [review]
Thank you

**Review by vcoolish — APPROVED** [review]

**Comment by doperiddle** [comment]
Why is it so hard to withdraw my assets n e were... I should be able to start living better

**Comment by doperiddle** [comment]
@doperiddle

**Pull request merged**